{"content": {"suites": {"0": {"status": {"total_pass": 0, "total_skip": 0, "total_xpass": 0, "total_xfail": 0, "total_rerun": 0, "total_fail": 1, "total_error": 0}, "tests": {"0": {"status": "FAIL", "message": "   socket.gaierror: [Errno -3] Temporary failure in name resolution\n       urllib3.exceptions.NameResolutionError: <urllib3.connection.HTTPSConnection object at 0x70ff5cdc5db0>: Failed to resolve 'googlechromelabs.github.io' ([Errno -3] Temporary failure in name resolution)\n       urllib3.exceptions.MaxRetryError: HTTPSConnectionPool(host='googlechromelabs.github.io', port=443): Max retries exceeded with url: /chrome-for-testing/latest-patch-versions-per-build.json (Caused by NameResolutionError(\"<urllib3.connection.HTTPSConnection object at 0x70ff5cdc5db0>: Failed to resolve 'googlechromelabs.github.io' ([Errno -3] Temporary failure in name resolution)\"))\n       requests.exceptions.ConnectionError: HTTPSConnectionPool(host='googlechromelabs.github.io', port=443): Max retries exceeded with url: /chrome-for-testing/latest-patch-versions-per-build.json (Caused by NameResolutionError(\"<urllib3.connection.HTTPSConnection object at 0x70ff5cdc5db0>: Failed to resolve 'googlechromelabs.github.io' ([Errno -3] Temporary failure in name resolution)\"))\n       requests.exceptions.ConnectionError: Could not reach host. Are you offline?\n", "test_name": "test_closePopUp", "rerun": "0"}}, "suite_name": "test_close_popup.py"}, "1": {"suite_name": "test_create_account.py", "tests": {"0": {"status": "FAIL", "message": "   selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {\"method\":\"xpath\",\"selector\":\"//*[@id=\"bs-6\"]/span/div/div/div/div/h4\"}\n     (Session info: chrome=127.0.6533.88); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception\n   Stacktrace:\n   #0 0x5ba453b5771a <unknown>\n   #1 0x5ba453828640 <unknown>\n   #2 0x5ba453877c0b <unknown>\n   #3 0x5ba453877ef1 <unknown>\n   #4 0x5ba4538bbb64 <unknown>\n   #5 0x5ba45389a90d <unknown>\n   #6 0x5ba4538b908a <unknown>\n   #7 0x5ba45389a683 <unknown>\n   #8 0x5ba45386ad71 <unknown>\n   #9 0x5ba45386b7de <unknown>\n   #10 0x5ba453b1f2ab <unknown>\n   #11 0x5ba453b23242 <unknown>\n   #12 0x5ba453b0c665 <unknown>\n   #13 0x5ba453b23dd2 <unknown>\n   #14 0x5ba453af12af <unknown>\n   #15 0x5ba453b46eb8 <unknown>\n   #16 0x5ba453b47090 <unknown>\n   #17 0x5ba453b564ec <unknown>\n   #18 0x7e7570c94ac3 <unknown>\n", "test_name": "test_create_account", "rerun": "0"}}, "status": {"total_pass": 0, "total_skip": 0, "total_xpass": 0, "total_xfail": 0, "total_rerun": 0, "total_fail": 1, "total_error": 0}}, "2": {"suite_name": "test_join_us.py", "tests": {"0": {"status": "FAIL", "message": "   selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {\"method\":\"xpath\",\"selector\":\"//*[@id=\"bs-6\"]/span/div/h1/span\"}\n     (Session info: chrome=127.0.6533.88); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception\n   Stacktrace:\n   #0 0x6057cf16a71a <unknown>\n   #1 0x6057cee3b640 <unknown>\n   #2 0x6057cee8ac0b <unknown>\n   #3 0x6057cee8aef1 <unknown>\n   #4 0x6057ceeceb64 <unknown>\n   #5 0x6057ceead90d <unknown>\n   #6 0x6057ceecc08a <unknown>\n   #7 0x6057ceead683 <unknown>\n   #8 0x6057cee7dd71 <unknown>\n   #9 0x6057cee7e7de <unknown>\n   #10 0x6057cf1322ab <unknown>\n   #11 0x6057cf136242 <unknown>\n   #12 0x6057cf11f665 <unknown>\n   #13 0x6057cf136dd2 <unknown>\n   #14 0x6057cf1042af <unknown>\n   #15 0x6057cf159eb8 <unknown>\n   #16 0x6057cf15a090 <unknown>\n   #17 0x6057cf1694ec <unknown>\n   #18 0x7dd646e94ac3 <unknown>\n", "test_name": "test_join_us_loggedOut", "rerun": "0"}, "1": {"status": "PASS", "message": "", "test_name": "test_join_us_login", "rerun": "0"}}, "status": {"total_pass": 1, "total_skip": 0, "total_xpass": 0, "total_xfail": 0, "total_rerun": 0, "total_fail": 1, "total_error": 0}}, "3": {"suite_name": "test_login.py", "tests": {"0": {"status": "FAIL", "message": "   selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {\"method\":\"xpath\",\"selector\":\"//*[@id=\"bs-6\"]/span/div/h1/span\"}\n     (Session info: chrome=127.0.6533.88); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception\n   Stacktrace:\n   #0 0x60db1411e71a <unknown>\n   #1 0x60db13def640 <unknown>\n   #2 0x60db13e3ec0b <unknown>\n   #3 0x60db13e3eef1 <unknown>\n   #4 0x60db13e82b64 <unknown>\n   #5 0x60db13e6190d <unknown>\n   #6 0x60db13e8008a <unknown>\n   #7 0x60db13e61683 <unknown>\n   #8 0x60db13e31d71 <unknown>\n   #9 0x60db13e327de <unknown>\n   #10 0x60db140e62ab <unknown>\n   #11 0x60db140ea242 <unknown>\n   #12 0x60db140d3665 <unknown>\n   #13 0x60db140eadd2 <unknown>\n   #14 0x60db140b82af <unknown>\n   #15 0x60db1410deb8 <unknown>\n   #16 0x60db1410e090 <unknown>\n   #17 0x60db1411d4ec <unknown>\n   #18 0x7e77d4a94ac3 <unknown>\n       AssertionError: Login failed\n       assert 1 == 0\n", "test_name": "test_login", "rerun": "0"}, "1": {"status": "PASS", "message": "", "test_name": "test_wrong_login", "rerun": "0"}}, "status": {"total_pass": 1, "total_skip": 0, "total_xpass": 0, "total_xfail": 0, "total_rerun": 0, "total_fail": 1, "total_error": 0}}, "4": {"suite_name": "test_logout.py", "tests": {"0": {"status": "FAIL", "message": "   selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {\"method\":\"xpath\",\"selector\":\"//*[@id=\"bs-6\"]/span/div/div/div/form/div[1]/input\"}\n     (Session info: chrome=127.0.6533.88); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception\n   Stacktrace:\n   #0 0x57e2a435a71a <unknown>\n   #1 0x57e2a402b640 <unknown>\n   #2 0x57e2a407ac0b <unknown>\n   #3 0x57e2a407aef1 <unknown>\n   #4 0x57e2a40beb64 <unknown>\n   #5 0x57e2a409d90d <unknown>\n   #6 0x57e2a40bc08a <unknown>\n   #7 0x57e2a409d683 <unknown>\n   #8 0x57e2a406dd71 <unknown>\n   #9 0x57e2a406e7de <unknown>\n   #10 0x57e2a43222ab <unknown>\n   #11 0x57e2a4326242 <unknown>\n   #12 0x57e2a430f665 <unknown>\n   #13 0x57e2a4326dd2 <unknown>\n   #14 0x57e2a42f42af <unknown>\n   #15 0x57e2a4349eb8 <unknown>\n   #16 0x57e2a434a090 <unknown>\n   #17 0x57e2a43594ec <unknown>\n   #18 0x7fb29f694ac3 <unknown>\n", "test_name": "test_logout", "rerun": "0"}}, "status": {"total_pass": 0, "total_skip": 0, "total_xpass": 0, "total_xfail": 0, "total_rerun": 0, "total_fail": 1, "total_error": 0}}, "5": {"suite_name": "test_myAccount.py", "tests": {"0": {"status": "FAIL", "message": "   selenium.common.exceptions.WebDriverException: Message: target frame detached: received Inspector.detached event\n     (failed to check if window was closed: disconnected: not connected to DevTools)\n     (Session info: chrome=127.0.6533.88)\n   Stacktrace:\n   #0 0x55921e67e71a <unknown>\n   #1 0x55921e34f640 <unknown>\n   #2 0x55921e338d9d <unknown>\n   #3 0x55921e337c26 <unknown>\n   #4 0x55921e33739f <unknown>\n   #5 0x55921e337292 <unknown>\n   #6 0x55921e3355ac <unknown>\n   #7 0x55921e335a9f <unknown>\n   #8 0x55921e34560b <unknown>\n   #9 0x55921e35b9a4 <unknown>\n   #10 0x55921e36103b <unknown>\n   #11 0x55921e3361ae <unknown>\n   #12 0x55921e35b4f1 <unknown>\n   #13 0x55921e3e02f3 <unknown>\n   #14 0x55921e3c1683 <unknown>\n   #15 0x55921e391d71 <unknown>\n   #16 0x55921e3927de <unknown>\n   #17 0x55921e6462ab <unknown>\n   #18 0x55921e64a242 <unknown>\n   #19 0x55921e633665 <unknown>\n   #20 0x55921e64add2 <unknown>\n   #21 0x55921e6182af <unknown>\n   #22 0x55921e66deb8 <unknown>\n   #23 0x55921e66e090 <unknown>\n   #24 0x55921e67d4ec <unknown>\n   #25 0x777558494ac3 <unknown>\n", "test_name": "test_myAccount", "rerun": "0"}}, "status": {"total_pass": 0, "total_skip": 0, "total_xpass": 0, "total_xfail": 0, "total_rerun": 0, "total_fail": 1, "total_error": 0}}, "6": {"suite_name": "test_reset_pwd.py", "tests": {"0": {"status": "FAIL", "message": "   selenium.common.exceptions.WebDriverException: Message: target frame detached: received Inspector.detached event\n     (failed to check if window was closed: disconnected: Unable to receive message from renderer)\n     (Session info: chrome=127.0.6533.88)\n   Stacktrace:\n   #0 0x58bc6eab071a <unknown>\n   #1 0x58bc6e781640 <unknown>\n   #2 0x58bc6e76ad9d <unknown>\n   #3 0x58bc6e769c26 <unknown>\n   #4 0x58bc6e76939f <unknown>\n   #5 0x58bc6e769292 <unknown>\n   #6 0x58bc6e7675ac <unknown>\n   #7 0x58bc6e767a9f <unknown>\n   #8 0x58bc6e77760b <unknown>\n   #9 0x58bc6e78d9a4 <unknown>\n   #10 0x58bc6e79303b <unknown>\n   #11 0x58bc6e7681ae <unknown>\n   #12 0x58bc6e78d4f1 <unknown>\n   #13 0x58bc6e8122f3 <unknown>\n   #14 0x58bc6e7f3683 <unknown>\n   #15 0x58bc6e7c3d71 <unknown>\n   #16 0x58bc6e7c47de <unknown>\n   #17 0x58bc6ea782ab <unknown>\n   #18 0x58bc6ea7c242 <unknown>\n   #19 0x58bc6ea65665 <unknown>\n   #20 0x58bc6ea7cdd2 <unknown>\n   #21 0x58bc6ea4a2af <unknown>\n   #22 0x58bc6ea9feb8 <unknown>\n   #23 0x58bc6eaa0090 <unknown>\n   #24 0x58bc6eaaf4ec <unknown>\n   #25 0x749a45294ac3 <unknown>\n       AssertionError: Error\n       assert 1 == 0\n", "test_name": "test_reset_pwd", "rerun": "0"}, "1": {"status": "FAIL", "message": "   selenium.common.exceptions.WebDriverException: Message: target frame detached: received Inspector.detached event\n     (failed to check if window was closed: disconnected: not connected to DevTools)\n     (Session info: chrome=127.0.6533.88)\n   Stacktrace:\n   #0 0x60f9a4d8971a <unknown>\n   #1 0x60f9a4a5a640 <unknown>\n   #2 0x60f9a4a43d9d <unknown>\n   #3 0x60f9a4a42c26 <unknown>\n   #4 0x60f9a4a4239f <unknown>\n   #5 0x60f9a4a42292 <unknown>\n   #6 0x60f9a4a4125d <unknown>\n   #7 0x60f9a4a664f1 <unknown>\n   #8 0x60f9a4aeb2f3 <unknown>\n   #9 0x60f9a4acc683 <unknown>\n   #10 0x60f9a4a9cd71 <unknown>\n   #11 0x60f9a4a9d7de <unknown>\n   #12 0x60f9a4d512ab <unknown>\n   #13 0x60f9a4d55242 <unknown>\n   #14 0x60f9a4d3e665 <unknown>\n   #15 0x60f9a4d55dd2 <unknown>\n   #16 0x60f9a4d232af <unknown>\n   #17 0x60f9a4d78eb8 <unknown>\n   #18 0x60f9a4d79090 <unknown>\n   #19 0x60f9a4d884ec <unknown>\n   #20 0x7ef56ac94ac3 <unknown>\n       AssertionError: Error\n       assert 1 == 0\n", "test_name": "test_wrng_reset_pwd", "rerun": "0"}}, "status": {"total_pass": 0, "total_skip": 0, "total_xpass": 0, "total_xfail": 0, "total_rerun": 0, "total_fail": 2, "total_error": 0}}}}, "date": "August 02, 2024", "start_time": 1722577756.251775, "total_suite": 7, "status": "FAIL", "status_list": {"pass": "2", "fail": "8", "skip": "0", "error": "0", "xpass": "0", "xfail": "0", "rerun": "0"}, "total_tests": "10"}